src/js/actions/DialogActions.js

```diff
@@ -0,0 +1,66 @@
+var AppDispatcher = require("../AppDispatcher");
```

Comment:
Then we need to refactor the `AppDispatcher.js` file to export a singleton `Dispatcher`, no?